doc/languages-frameworks/vim.section.md

```diff
@@ -33,55 +36,51 @@ vim-full.customize {
 }
 ```
 
-This configuration is used when Vim is invoked with the command specified as name, in this case `vim-with-plugins`.
-You can also omit `name` to customize Vim itself. See the
-[definition of `vimUtils.makeCustomizable`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vim/plugins/vim-utils.nix#L408)
+This configuration is used when Vim is invoked with the command specified as
+name, in this case `vim-with-plugins`. You can also omit `name` to customize
+Vim itself. See the [definition of
+`vimUtils.makeCustomizable`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vim/plugins/vim-utils.nix#L408)
 for all supported options.
 
 
 ## Managing plugins with Vim packages {#managing-plugins-with-vim-packages}
 
-To store your plugins in Vim packages (the native Vim plugin manager, see `:help packages`) the following example can be used:
+To store your plugins in Vim packages (the native Vim plugin manager, see
+`:help packages`) the following example can be used:
```

Comment:
This seems to be an unrelated formatting change for the docs.

1. If you want to reformat docs, that should be its own PR. Having them here makes it much more difficult to find the _actual_ changes when reviewing.
2. Nixpkgs prefers [One sentence per line](https://github.com/NixOS/nixpkgs/blob/master/doc/README.md#one-sentence-per-line), not wrapping at a line-length.

Same applies elsewhere in docs.

Perhaps you ran a format-on-save tool that's incompatible with nixpkgs' formatting guidelines?